core/src/main/java/com/bettercloud/bigtable/orm/BigTableEntityDao.java

```diff
@@ -311,4 +322,43 @@
 
         table.delete(deletes);
     }
+
+    private T delegateToEntity(final Result result) throws IOException {
```

Comment:
I don't like this method name since there's nothing to delegate _to_ until it's invoked. `convertToEntity` is probably more appropriate.

Reply:
renamed to `convertToEntity`